docs/developers/community.md

```diff
@@ -22,6 +22,17 @@ Github is where the work gets done. It's the best place for technical conversati
 | [Contracts](https://github.com/farcasterxyz/contracts) | An implementation of the Farcaster Contracts |
 | [Hubble](https://github.com/farcasterxyz/hub-monorepo) | An implementation of a Farcaster Hub         |
 
+## Community SDKs
+
+This section references the language specific Farcaster community built SDKs for interacting hubs directly, or the warpcast APIs.
+
+| Language                                                      | Description                               |
+| ------------------------------------------------------------- | ----------------------------------------- |
```

Comment:
these are all warpcast api sdks, not farcaster sdks so i think we should remove this section

Reply:
I tried to tune this around only hubs, but man, it feels so overwhelming sometimes that I felt compelled...

Maybe I'll play around with a link to the awesome-farcaster list